### ISSUE TITLE
Bug: Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: @eslint\eslintrc\universal.js

### ISSUE THREAD

**Issue by brendonco** [eslint/eslint#15247]
### Environment

Node version: 12.16.0
npm version: 6.13.4
Local ESLint version: 8.1.0
Global ESLint version: 8.1.0
Operating System: win 10

### What parser are you using?

Other

### What did you do?

<details>
<summary>Configuration</summary>

```
{
  "extends": [
    "airbnb",
    "plugin:jest/recommended",
    "plugin:testing-library/react",
    "plugin:jest-dom/recommended",
    "prettier"
  ],
  "parser": "babel-eslint",
  "env": {
    "browser": true,
    "node": true,
    "jasmine": true
  },
  "rules": {
    "import/no-named-as-default": 0,
    "linebreak-style": 0,
    "arrow-body-style": 0,
    "no-confusing-arrow": 0,
    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
    "import/prefer-default-export": 0,
    "react/jsx-boolean-value": 0,
    "react/sort-comp": 0,
    "jsx-a11y/no-static-element-interactions": 0,
    "class-methods-use-this": 0,
    "react/no-array-index-key": 0,
    "react/jsx-no-bind": 0,
    "react/jsx-indent": 0,
    "react/no-unused-state": 0,
    "react/prefer-stateless-function": [1],
    "no-mixed-operators": 0,
    "no-shadow": "off",
    "jsx-a11y/anchor-is-valid": [
      "error",
      {
        "components": ["Link"],
        "aspects": ["invalidHref", "preferButton"]
      }
    ],
    "jsx-a11y/click-events-have-key-events": 0,
    "jsx-a11y/no-noninteractive-element-interactions": 0,
    "jsx-a11y/heading-has-content": 0,
    "jsx-a11y/label-has-for": 0,
    "indent": ["error", 2, { "SwitchCase": 1 }],
    "import/extensions": ["error", "never"],
    "react/jsx-props-no-spreading": 0,
    "import/no-extraneous-dependencies": ["error", { "devDependencies": true }],
    "jsx-a11y/label-has-associated-control": [
      2,
      {
        "controlComponents": ["SelectField"],
        "depth": 3
      }
    ],
    "react-hooks/rules-of-hooks": "error",
    "react-hooks/exhaustive-deps": "warn"
  },
  "plugins": ["jest", "react-hooks", "testing-library", "jest-dom"],
  "settings": {
    "import/resolver": "webpack"
  }
}

```
</details>

```js
not even checking js codes
```

```bash
Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: \node_modules\@eslint\eslintrc\universal.js
require() of ES modules is not supported.
require() of \node_modules\@eslint\eslintrc\universal.js from \node_modules\eslint\lib\linter\linter.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename universal.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from \node_modules\@eslint\eslintrc\package.json.
at Object.Module._extensions..js (internal/modules/cjs/loader.js:1173:13)
    at Module.load (internal/modules/cjs/loader.js:1001:32)
    at Function.Module._load (internal/modules/cjs/loader.js:900:14)
    at Module.require (internal/modules/cjs/loader.js:1043:19)
    at require \node_modules\v8-compile-cache\v8-compile-cache.js:159:20)
    at Object.<anonymous>
```

### What did you expect to happen?

to lint successful.  Downgrade eslint to 7.27.0 works.

### What actually happened?

linter failed

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by aladdin-add** [comment]
to use eslint v8, please try to upgrade node.js, as it requires "^12.22.0 || ^14.17.0 || >=16.0.0".
